micropython/senml/examples/actuator.py

```diff
@@ -0,0 +1,68 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2018 KPN (Jan Bogaerts)
+Copyright (c) 2023 MicroPython
```

Comment:
For this copyright line, please use either your name or Arduino company name (not "MicroPython").